bin/webpack.js

```diff
@@ -140,12 +139,30 @@ yargs.options({
 		type: 'boolean',
 		group: DISPLAY_GROUP,
 		describe: 'Show more details'
+	},
+	'stats': {
+		type: 'string',
+		describe: 'Display information using a preset(Accepted presets: none, errors-only, minimal, normal, verbose)',
```

Comment:
We should try to make this better looking in the terminal. Shorten the text, and make the options be inline. Should also split this string to two lines